### DISCUSSION TITLE
System Tray Icons Not Displaying

### DISCUSSION THREAD

**Discussion by RyanATX** [valinet/ExplorerPatcher#3950] [Q&A] [answered]
![system_tray_icons](https://github.com/user-attachments/assets/2e3b79f0-f984-4cca-8e45-1d6389a431bc)

Despite the settings to turn these icons on, they are not showing in the System Tray. Sometimes the envelope will show when I get a new e-mail but will disappear (even if I haven't opened the e-mail or Outlook at all). 

Any ideas?

Thanks for your time.

**Comment by milestprower92** [comment]
Open your start/search menu and type Properties (ExplorerPatcher) and open it, then click on 'Customize notification area icons'

**Reply by RyanATX** [reply]
![follow-up](https://github.com/user-attachments/assets/56ecd1a1-5bb5-4a9b-809b-fe77dd456f17)

Thanks for the quick response. I thought I had clicked on everything. 

I changed the settings (as shown in the screen shot) and clicked "OK" but they don't change. When I go back into the settings they are all reset back to "Only show notifications." Have restarted after making the changes and same issue.

Am I missing something?

Thanks again.

**Reply by RyanATX** [reply]
I guess I accidently figured it out. Neither the Windows or ExplorerPatcher settings made them appear. But I was able to drag them from the "show hidden icons" into the system tray. I stumbled upon this by accidently removing one.

![image](https://github.com/user-attachments/assets/f4e38239-fd7f-40d7-a29e-e561d862ce40)

**Reply by pyrates999** [reply, marked as answer]
not being able to customize the system tray icons is mentioned here: https://github.com/valinet/ExplorerPatcher/issues/1454

So you can drag and drop them.  You can also just have it set to show all of them.
